framework/src/mesh/MooseMesh.C

```diff
@@ -140,6 +140,14 @@ validParams<MooseMesh>()
                         " when DistributedMesh is used. Value is ignored in ReplicatedMesh mode");
   params.addParam<unsigned int>(
       "patch_size", 40, "The number of nodes to consider in the NearestNode neighborhood.");
+  params.addParam<bool>("automatic_patch_update",
+                        false,
+                        "Updates the patch for slave nodes for which penetration is not detected.");
```

Comment:
It seems to me that this should default to true.

Reply:
I will change this to true and check if all the tests pass. Setting it to true for all problems might increase run times.